tools/deb/envoy_bootstrap_tmpl.json

```diff
@@ -33,10 +38,10 @@
     "deprecated_v1": {
       "sds_config": {
         "api_config_source": {
-          "api_type": "REST_LEGACY",
+          "api_type": "GRPC",
           "refresh_delay": {{ .refresh_delay }},
           "cluster_names": [
-            "rds"
+            "eds-grpc"
```

Comment:
change it to xds-grpc, not specific to eds.

Reply:
+1 - but can be done in the next PR.